ngraph/frontend/onnx/onnx_import/include/onnx_import/utils/onnx_internal.hpp

```diff
@@ -36,8 +36,25 @@ namespace ngraph
             ///
             /// \return     An nGraph function that represents a single output from the created
             /// graph.
-            std::shared_ptr<Function> import_onnx_model(ONNX_NAMESPACE::ModelProto& model_proto,
-                                                        const std::string& model_path);
+            ONNX_IMPORTER_API
+            std::shared_ptr<Function>
+                import_onnx_model(std::shared_ptr<ONNX_NAMESPACE::ModelProto> model_proto,
```

Comment:
I believe it's prohibited to export any function with protobuf entities.

Reply:
since `importer` and 'editor' was merged, it seems to not needed